<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignInWindow</class>
 <widget class="QWidget" name="SignInWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="WidgetBtn" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="SignInBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{   
    background-color: rgba(128, 128, 128, 20);
    color:rgb(57, 58, 58);
}

/* 鼠标悬停 */
QPushButton:hover{
	color: rgb(205, 207, 207);
}</string>
      </property>
      <property name="text">
       <string>登陆</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BackBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{   
    background-color: rgba(128, 128, 128, 20);
    color:rgb(57, 58, 58);
}

/* 鼠标悬停 */
QPushButton:hover{
	color: rgb(205, 207, 207);
}</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="WidgetKey" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}</string>
      </property>
      <property name="text">
       <string>管理员密钥：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 234, 234);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="appName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{   
    border-top: 3px outset transparent;          /*边框全透明*/
	border-bottom: 3px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;  
    min-width: 0px;                            /* 控件最小宽度 */
    min-height: 0px;
    border-radius: 15px;               /*圆角*/
	background-color: rgba(0, 0, 0,0);
	color: rgb(57, 58, 58);
}</string>
   </property>
   <property name="text">
    <string>交通查询系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="appPic">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255,255,255,0)</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
